packages/faust-nx/src/components/WordPressTemplate.tsx

```diff
@@ -1,13 +1,23 @@
-import React, { PropsWithChildren } from 'react';
+import React, { PropsWithChildren, ReactNode } from 'react';
 import { DocumentNode, useQuery } from '@apollo/client';
 import { getTemplate } from '../getTemplate.js';
 import { SeedNode } from '../queries/seedQuery.js';
 import { getConfig } from '../config/index.js';
+import { WordPressTemplate as WordPressTemplateType } from '../getWordPressProps';
 
 export type WordPressTemplateProps = PropsWithChildren<{
   __SEED_NODE__: SeedNode;
 }>;
 
+function cleanTemplate(
+  template: WordPressTemplateType,
+): React.FC<{ [key: string]: any }> {
+  const copy = template;
+  delete copy.query;
```

Comment:
clean up those properties before rendering them in React.

Reply:
Nice detail 💯